src/helpers/sort-sizes.ts

```diff
@@ -0,0 +1,38 @@
+const sortOrder = [
+	"3.0",
+	"3.5",
+	"4.0",
+	"4.5",
+	"5.0",
+	"5.5",
+	"6.0",
+	"6.5",
+	"7.0",
+	"7.5",
+	"8.0",
+	"8.5",
+	"9.0",
+	"9.5",
+	"10.0",
+	"10.5",
+	"11.0",
+	"11.5",
+	"4XS",
+	"3XS",
+	"XXS",
+	"XS",
+	"S",
+	"M",
+	"L",
+	"XL",
+	"XXL",
+	"XXXL",
+];
```

Comment:
move to constants (folder and file). no one will find it here

Reply:
I've moved const sortOrder to constants, but left function and folder. we have already another files with helpers functions in this folder. it is our structure 
![image](https://github.com/ZenBit-Tech/sunny-bunnies_fe/assets/120641395/6054e668-98dc-4cd6-942d-32ab2f66d8db)
if it's not good I'll change